openslides/mediafiles/models.py

```diff
@@ -44,7 +44,8 @@ class Meta:
             ('can_see', 'Can see the list of files'),
             ('can_see_hidden', 'Can see hidden files'),
             ('can_upload', 'Can upload files'),
-            ('can_manage', 'Can manage files'))
+            ('can_manage', 'Can manage files'),
+            ('can_manage_logos', 'Can manage logos'))
```

Comment:
This perm isn't used on server side, is it? Someone with 'can_manage_logos' but without 'can_manage_config' get errors?!